.importlinter

```diff
@@ -14,4 +14,6 @@ layers=
     adapters
     application
     domain
-exhaustive = true
\ No newline at end of file
+exhaustive = true
+exhaustive_ignores =
```

Comment:
Rather than make an exception of this module, can we just list it as the top layer?